test_automation/features/steps/login.py

```diff
@@ -0,0 +1,49 @@
+import features.steps.storage as storage
+import features.steps.login_selectors as login_selectors
+
+from features.steps.common_actions import short_wait
+
+from behave import given, step, then, when
+from behave.runner import Context
+
+
+@given("I have url")
+def get_form_url(context: Context) -> None:
+    context.browser.get(storage.form_url)
+
+
+@given("I am on login page")
+def login_page(context: Context) -> None:
+    context.browser.find_element(*login_selectors.main_logo)
+    context.browser.find_element(*login_selectors.login_body)
+    context.browser.find_element(*login_selectors.languages_container)
+
+
+@step('I input "{username}" username')
+def input_username(context: Context, username: str) -> None:
+    username_field = context.browser.find_element(*login_selectors.username_field)
+    username_field.click()
+    username_field.send_keys(username)
+
+
+@step('I input "{password}" password')
+def input_password(context: Context, password: str) -> None:
+    password_field = context.browser.find_element(*login_selectors.password_field)
+    password_field.click()
+    password_field.send_keys(password)
+
+
+@when("I click login button")
+def click_login(context: Context) -> None:
+    login_button_enabled = context.browser.find_element(*login_selectors.login_button_enabled)
+    login_button_enabled.click()
```

Comment:
`input_username()`, `input_password()` and `click_login()` look very much the same to me - that's a good opportunity to take the repeated code out to another helper function, somewhere in `common_actions`.
Here's a rough draft of that:
```
# common_actions.py
def click_on(context: Context, element: Tuple[str, str]) -> WebElement:
    return context.browser.find_element(*element).click()
```
```
# login.py
@step('I input "{password}" password')
def input_password(context: Context, password: str) -> None:
    click_on(*login_selectors.password_field).send_keys(password)
```

It looks like the `find_element()/click()` combo is the default for pretty much every step it would be good to have it shortened to side-function helper, just like the `short_wait()` function.